resources/js/composables/photo/basePhoto.ts

```diff
@@ -57,27 +67,32 @@ export function usePhotoBaseFunction(
 		return "width: " + photo.value?.size_variants.original.width + "px; height: " + photo.value?.size_variants.original.height + "px";
 	});
 
-	const imageViewMode = computed(() => {
+	const imageViewMode = computed<ImageViewMode>(() => {
 		if (photo.value?.precomputed.is_video) {
-			return 0;
+			return ImageViewMode.Video;
 		}
+
 		if (photo.value?.precomputed.is_raw) {
 			if (photo.value?.size_variants.medium !== null) {
-				return 2;
+				return ImageViewMode.Medium;
 			}
-			return 1;
+			if (photo.value?.size_variants.original?.url?.endsWith(".pdf")) {
+				return ImageViewMode.Pdf;
+			}
+			return ImageViewMode.Raw;
 		}
 
-		if (!photo.value?.precomputed.is_livephoto) {
+		if (photo.value?.precomputed.is_livephoto === true) {
```

Comment:
Is there a reason why you explicitly wrote === true here?

Reply:
I flipped the conditions and wanted to be exact in the behaviour. I prefer clarity even thought it might be too much in this case.